src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,16 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ post }) => (
+  <>
+    {post.comments.length !== 0 ? (
+      <div className="CommentList">
+        {post.comments.map(comment => (
+          <CommentInfo comment={comment} key={comment.id} />
+        ))}
+      </div>
+    ) : (
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    )}
```

Comment:
you lose "<hr />" before line 13
please fix it, because you'd better follow to mockup